docs/source/config.rst

```diff
@@ -18,8 +18,9 @@ All datalad-specific configuration variables are prefixed with ``datalad.``.
 It is possible to override or amend the configuration using environment
 variables. Any variable with a name that starts with ``DATALAD_`` will
 be available as the corresponding ``datalad.`` configuration variable,
-replacing any ``_`` in the name with a dot, and all letters converted
-to lower case. Values from environment variables take precedence over
+replacing any ``__`` (two underscores) with a hyphen, then any ``__``
```

Comment:
I think the second one is also two underscores not just one. IMHO good to fix and push directly to master now that typos are caught ;-)

Reply:
I will just do it ... cheers.

Reply:
Good eyes :]  Thanks